test/system/500-networking.bats

```diff
@@ -139,6 +139,9 @@ load helpers
                    $IMAGE nc -l -n -v -p $myport
         cid="$output"
 
+        # Workaround: Process execution, Socket & port binding isn't instant.
+        sleep 1s
```

Comment:
Suggestion: copy and use https://github.com/containers/podman/blob/0a0ade3cc00c3779bbf68ddd103d3efd10b5c25b/test/apiv2/test-apiv2#L439-L454

Reply:
Oh! Okay, that's better, thanks!

Reply:
I tweaked the `die` message slightly, before adding it into `helpers.bash`.